_layouts/assignment.html

```diff
@@ -3,7 +3,7 @@
 sections: mp-content
 ---
 
-< script src='https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/latest.js?config=TeX-MML-AM_CHTML' defer></script >
+<script src='https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/latest.js?config=TeX-MML-AM_CHTML' defer></script >
```

Comment:
```suggestion
<script src='https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/latest.js?config=TeX-MML-AM_CHTML' defer></script>
```

Reply:
this is also broken in the [mp1 branch](https://github.com/illinois-cs498vr/illinois-cs498vr.github.io/pull/178)